python/pyarrow/rados.py

```diff
@@ -24,12 +24,33 @@
 
 
 class SplittedParquetWriter(object):
+    """
+    Write data to a file of RadosParquetFile format.
+
+    Parameters
+    ---------
+    filename: The name of the file to write to.
```

Comment:
here the filename means the name of the file to split, not the ones to write to.

Reply:
Done.